package step01.exam05;

import java.util.Scanner;

public class Exam01 {

	public static void main(String[] args) {
		Scanner keyScan = new Scanner(System.in);	
		int a,b;
		while(true) {
			System.out.println("a/b?");
			a= keyScan.nextInt();
			b= keyScan.nextInt();
			System.out.printf("결과는 %d입니다.",a/b);
			//0으로 나누게되면 예외가 발생한다.
			//=>예외가 발생하면 실행을 멈추고 이 메소드를 호출 한 쪽에  즉시 보고한다.
			//=>main()를 호출한 것은 JVM이다.
			//=>JVM은 예외를 보고 받으면 즉시 실행을 멈춘다.
			//바로 이것이 문제!
					
		}

	}

}
